core/src/main/java/io/questdb/cairo/TableReader.java

```diff
@@ -440,7 +440,7 @@ public double sumDouble(int columnIndex) {
                 for (int pageIndex = 0, pageCount = column.getPageCount(); pageIndex < pageCount; pageIndex++) {
                     long a = column.getPageAddress(pageIndex);
                     long count = column.getPageSize(pageIndex) / Double.BYTES;
-                    result += Vect.sumDouble(a, count);
+                    result += Vect.sumDoubleNeumaier(a, count);
```

Comment:
could we roll this line back and then we can commit C-level impls that can be integrated into SQL later